elide-graphql/src/test/resources/graphql/requests/fetch/fragmentInline.graphql

```diff
@@ -0,0 +1,33 @@
+query namedQuery {
+  author {
+    ...AuthorFields
+  }
+}
+
+fragment AuthorFields on author {
+  edges {
+    node {
+      id
+      books {
+        ... on book {
+          ...BookFields
```

Comment:
Isn't this another loop test? How is this different than the other loop failure test?

Reply:
It would break at the inline fragment first.